### DISCUSSION TITLE
Deploy stored procedures with git

### DISCUSSION THREAD

**Discussion by emschwartz** [supabase/supabase#1979] [Questions] [answered]
As nice as the SQL editor is, it would probably be better to save stored procedures in git and have a Github action load them into the DB.

Does anyone have a good template for doing this? (Could / should something like this be added to the docs?)

**Comment by silentworks** [comment, marked as answer]
You can use a CLI tool to deal with your database migrations and stored procedures. I personally use [ley](https://github.com/lukeed/ley) for all my DB migrations. This can be easily setup with something like Github actions to run.
